files-community/Files
# Fix: Fixed issue where folders with dots in their name couldn't be opened if compressed

**Resolved / Related Issues**
   Closes #11567 

**Validation**
How did you test these changes?
   1. Open a zip folder containing a folder
   2. Try to open the folder (**Important**: Layout must be set to `Details`, since with others it was already working)
